docs/concepts/bitswap.md

```diff
@@ -13,23 +13,23 @@ related:
 Bitswap is a core module of IPFS for exchanging blocks of data. It directs the requesting and sending of blocks to and from other peers in the network. Bitswap is a _message-based protocol_ where all messages contain [wantlists](#wantlist) or blocks. Bitswap has a [Go implementation](https://github.com/ipfs/go-bitswap) and a [JavaScript implementation](https://github.com/ipfs/js-ipfs-bitswap).
```

Comment:
Looks like I missed a _wantlist_ to _want-list_ change here, but can no longer edit.